website/content/api-docs/index.mdx

```diff
@@ -75,6 +75,20 @@ $ curl \
     http://127.0.0.1:8500/v1/kv/foo
 ```
 
+## Parses URL-Encoded Resource Names
```

Comment:
I chose the word "Parses" rather than "Accepts" because Consul may still reject the request if the resource name is not valid for that endpoint.